src/lib/registration/ui/base_system_registration_dialog.rb

```diff
@@ -65,6 +65,9 @@ def run
           Yast::GetInstArgs.enable_next || Yast::Mode.normal
         )
 
+        # Limit the reg_code InputField to 512 (bsc#1098576)
```

Comment:
well ... a bit useless comment ;-)

Reply:
Actually, it "explains" why that control was introduced, linking to the associated bug. As far as I have understood until now (reading the code), this is a quite common practice in YaST. Isn't it?

If you want, I could reword it to something like `Avoid sending too large request limiting the registration code (bsc#1098576)` or whatever other reasonable suggestion :)

Reply:
I'm fine with the current comment, the most important part is the bug number which is present.